src/Hooks.ts

```diff
@@ -16,9 +16,14 @@ export function useNavigation<S>(): NavigationScreenProp<S & NavigationRoute> {
 }
 
 export function useNavigationParam<T extends keyof NavigationParams>(
-  paramName: T
+  paramName: T,
+  fallbackValue?
```

Comment:
I think this should be

```suggestion
  fallbackValue?: NavigationParams[T]
```

Reply:
The I get this error in line 24:
```
(parameter) fallbackValue: NavigationParams[T] | undefined
Argument of type 'NavigationParams[T] | undefined' is not assignable to parameter of type 'NonNullable<NavigationParams[T]>'.
  Type 'undefined' is not assignable to type 'NonNullable<NavigationParams[T]>'.ts(2345)
```

what works is:
```javascript
export function useNavigationParam<T extends keyof NavigationParams>(
  paramName: T,
  fallbackValue?: NavigationParams[T]
) {
  const { getParam, setParams } = useNavigation()
  return [
    fallbackValue? getParam(paramName, fallbackValue): getParam(paramName),
    (newValue: NavigationParams[T]) => setParams({ [paramName]: newValue }),
  ]
}
```
but that ternary seems kind of ugly, is there a more elegant solution?

Reply:
Hmm, looks like it's an issue with the typings of `getParam`. In that case I think ternary is reasonable